R/formula.R

```diff
@@ -164,3 +166,73 @@ f_list <- function(...) {
 as_f_list <- function(x) {
   .Call(lhs_name, x)
 }
+
+#' Coerce quoted expressions to quoted formula.
```

Comment:
I'm not sure about the name of this function, but everything else looks ok, so I'd say merge.

This may be another candidate to live in the lazyeval package, since it's in support of a specific way of handling NSE/quoted args